nextx/settings.py

```diff
@@ -0,0 +1,9 @@
+from pydantic import BaseSettings
+from os import environ as env
+
+
+class Settings(BaseSettings):
+    redis_host: str = env.get("REDIS_HOST", "localhost")
+
+
```

Comment:
I believe that pydantic already comes with support for .env variables, just need to define the name. I'll check, nice work